python/servo/package_commands.py

```diff
@@ -96,8 +96,11 @@ def copy_dependencies(binary_path, lib_path):
 
 def copy_windows_dependencies(binary_path, destination):
     deps = [
+        "api-ms-win-crt-runtime-l1-1-0.dll",
         "libcryptoMD.dll",
         "libsslMD.dll",
+        "msvcp140.dll",
+        "vcruntime140.dll",
```

Comment:
It's less the size, and more if we have the license to distribute them.  For two of the three files, we are granted permission, but can you link where we're granted permission to redist `api-ms-win-crt-runtime-l1-1-0.dll`?